src/design/crosslink/construction.md

```diff
@@ -74,6 +74,8 @@ A Crosslink node must participate in <span style="white-space: nowrap">both $\Pi
 
 A player’s view in $\Pi_{\mathrm{*bft}}$ includes a set of <span style="white-space: nowrap">$\mathrm{*}$bft‑block chains</span> each rooted at a fixed genesis <span style="white-space: nowrap">$\mathrm{*}$bft‑block $\mathcal{O}_{\mathrm{*bft}}$.</span> There is a <span style="white-space: nowrap">$\mathrm{*}$bft‑block‑validity</span> rule (specified below), which depends only on the content of the block and its ancestors. A non‑genesis block can only be <span style="white-space: nowrap">$\mathrm{*}$bft‑block‑valid</span> if its parent is $\mathrm{*}$bft‑block‑valid. <span style="white-space: nowrap">A $\mathrm{*}$bft‑valid‑chain</span> is a chain of <span style="white-space: nowrap">$\mathrm{*}$bft‑block‑valid</span> blocks.
 
+Execution proceeds in a sequence of epochs. In each epoch, a <span style="white-space: nowrap">$\mathrm{*}$bft‑proposal</span> may be made.
```

Comment:
In review we realized this text is misleading, since malicious or confused proposers may make multiple proposals, or some protocols may select multiple proposers, then find consensus downstream of the proposal creation.